packages/@react-aria/dnd/stories/VirtualizedListBox.tsx

```diff
@@ -196,26 +202,26 @@ export const VirtualizedListBox = React.forwardRef(function (props: any, ref) {
 });
 
 function CollectionItem({item}) {
-  let {state, dropState} = React.useContext(Context);
-  let ref = React.useRef(undefined);
+  let {state, dropState} = React.useContext(Context) ?? {};
```

Comment:
```suggestion
  let {state, dropState} = React.useContext(Context)!;
```

you can avoid the later `!` on `state` and `dropState` then